js-apps/meep-frontend/src/js/util/scenario-utils.js

```diff
@@ -319,8 +325,12 @@ export function addElementToScenario(scenario, element) {
     scenarioElement = createZone(uniqueId, name, element);
     break;
   }
+  case ELEMENT_TYPE_POA_CELL_4G: {
+    scenarioElement = createPoaCell4g(uniqueId, name, element);
+    break;
+  }
   case ELEMENT_TYPE_POA: {
-    scenarioElement = createNL(uniqueId, name, element);
+    scenarioElement = createPoa(uniqueId, name, element);
```

Comment:
For consistency with other elements, suggest keeping createNL() and passing a type string as a parameter.

Reply:
i kept the Poa, PoaCell, Domain, DomainCell rather than using a createDomain() or createNL() with parameter since the content of the structures are different and i felt it was better then using if/else inside. Please comment if you don't agree.